aws/src/test/java/org/apache/iceberg/aws/s3/TestS3FileIO.java

```diff
@@ -447,7 +448,7 @@ private void createRandomObjects(String prefix, int count) {
   private void createBucket(String bucketName) {
     try {
       s3.get().createBucket(CreateBucketRequest.builder().bucket(bucketName).build());
-    } catch (BucketAlreadyExistsException e) {
+    } catch (BucketAlreadyExistsException | BucketAlreadyOwnedByYouException e) {
```

Comment:
@nastra this seems unrelated?

Reply:
this is actually required to make the tests pass

Reply:
the newer version of the API throws a new exception which the tests most likely just want to ignore

Reply:
Ah I see, thanks 👍